core/state/snapshot/generate.go

```diff
@@ -466,6 +476,11 @@ func (dl *diskLayer) generateRange(root common.Hash, prefix []byte, kind string,
 		internal time.Duration
 	)
 	nodeIt.AddResolver(snapNodeCache)
+
+	var r *danglingRange
```

Comment:
Please rename `r` to something meaningful

Reply:
fixed